PredixMobileUIKit/PredixBarChartView.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  PredixBarChartView.swift
+//  PredixMobileUIKit
+//
+//  Created by Fouche, George (GE Healthcare) on 11/15/17.
+//  Copyright © 2017 GE. All rights reserved.
+//
+
+import Charts
+import Foundation
+import PredixSDK
+
+/// The Bar class
+public class Bar {
+    var yValues: [Double]
+    var label: String
+    var color: [UIColor]?
```

Comment:
Avoid optional arrays, generally you'd want to initialize to an empty array.
What is the behavior if there is no color array? Should be documented.

Reply:
Optional arrays have been removed